### PR TITLE
Cidr 1375

### PR DESCRIPTION
**Pull request by KJStick**

### CODE REVIEW

File: wxt_cidrbot/cmd_list.py [hunk 1]
```diff
@@ -341,6 +355,14 @@ def message_similarity(self, text_split, word_list, msg_threshold):
                         likely_words.append(key_word)
         return likely_words
 
+    def verify_batch_repos(self, repos):
```
Comment (jlothian) [inline wxt_cidrbot/cmd_list.py:358]: why is this function needed?

File: wxt_cidrbot/git_webhook_handler.py [hunk 2]
```diff
@@ -108,6 +110,15 @@ def webhook_request(self, event):
             elif event_action == 'review_requested':
                 self.send_review_message(installation_id, json_string, False, None)
 
+        elif x_event_type == 'pull_request_review':
+            state = json_string['review']['state']
+            state = state.lower()
+
+            self.logging.debug("JSON DATA: %s", json.dumps((event["body"])))
```
Comment (jlothian) [inline wxt_cidrbot/git_webhook_handler.py:117]: Just checking - there's nothing in this data like password, tokens, credentials, etc. that shouldn't be logged, right?

File: wxt_cidrbot/cmd_list.py (hunk 1 above)
Comment (ppajersk) [reply wxt_cidrbot/cmd_list.py:358]: This was something I suggested so it made the code more concise by extracting this functionality. It basically just applies regex to the repos.

**Review by ppajersk — DISMISSED** [review]
Looks good

**Review by ppajersk — DISMISSED** [review]
Test approval

**Review by ppajersk — DISMISSED** [review]
Look's like the message is fixed now, good to go

File: wxt_cidrbot/git_webhook_handler.py [hunk 3]
```diff
@@ -325,6 +333,95 @@ def send_review_message(self, installation_id, json_string, codeowners_status, r
...
+        check_runs_url = f"https://api.github.com/repos/{repo_name}/commits/{branch_name}/check-runs"
+        check_runs_json = session.get(check_runs_url, headers={}).json()
+
+        passed_check_runs = True
+        for run in check_runs_json['check_runs']:
+            if run['conclusion'].lower() != 'success':
+                passed_check_runs = False
+                break
+
+        reviews_mark = self.EMJOIS['GREEN_CHECK']
```
Comment (ppajersk) [inline wxt_cidrbot/git_webhook_handler.py:400]: Emjois is miss spelled

**Review by ppajersk — CHANGES_REQUESTED** [review]

**Review by ppajersk — APPROVED** [review]
Approved

**Pull request merged**